src/components/organisms/UiTabs/UiTabs.stories.ts

```diff
@@ -0,0 +1,337 @@
+    contentAttrs: { 'data-testid': 'search-content' },
+  },
+  {
+    name: 'point',
+    title: 'Point on the body',
+  },
+  {
+    name: 'drop',
+    title: 'Drop body point',
+  },
+];
+const content = {
+  search: 'Serum uric acid concentration',
+  point: '1. Erythrocyte Sedimentation Rate',
+};
+const modifiers = '';`;
+
+const basicCode = `<template>
+  ${basicTemplate}
+</template>
+
+<script setup lang="ts">
+${basicScript}
+</script>
+`;
+
+const fixedScript = `import { ref } from 'vue';
+import { UiTabs, UiText } from '@infermedica/component-library';
+
+const modelValue = ref('point');
+const items = [
+  {
+    name: 'search',
+    title: 'Search',
+    'data-testid': 'search',
+    buttonTabAttrs: { 'data-testid': 'search-button' },
+    contentAttrs: { 'data-testid': 'search-content' },
+  },
+  {
+    name: 'point',
+    title: 'Point on the body',
+  },
+  {
+    name: 'drop',
+    title: 'Drop body point',
+  },
+];
+const content = {
+  search: 'Serum uric acid concentration',
+  point: '1. Erythrocyte Sedimentation Rate',
+};
+const modifiers = 'ui-tabs-fixed';`;
+
+const fixedCode = `<template>
+  ${basicTemplate}
+</template>
+
+<script setup lang="ts">
+${fixedScript}
+</script>
+`;
+
+const defaultSlotTemplate = `<UiTabs
+    v-model="modelValue"
+    :class="modifiers"
+    >
+    <template
+      v-for="(item, key) in items"
+      :key="key"
+    >
+      <UiTabsItem
+        :title="item.title"
+        :name="item.name"
+        :button-attrs="item.buttonTabAttrs"
+        :content-attrs="item.contentAttrs"
+        v-bind="{
+          'data-testid': item['data-testid'],
+        }"
+      >
+        <UiText>
+          {{ content[item.name] }}
+        </UiText>
+      </UiTabsItem>
+    </template>
+  </UiTabs>`;
+
+const defaultSlotScript = `import { ref } from 'vue';
+import { UiTabs, UiTabsItem, UiText } from '@infermedica/component-library';
+import UiTabsItem from '@infermedica/component-library';
+
+const modelValue = ref('point');
+const items = [
+  {
+    name: 'search',
+    title: 'Search',
+    'data-testid': 'search',
+    buttonTabAttrs: { 'data-testid': 'search-button' },
+    contentAttrs: { 'data-testid': 'search-content' },
+  },
+  {
+    name: 'point',
+    title: 'Point on the body',
+  },
+  {
+    name: 'drop',
+    title: 'Drop body point',
+  },
+];
+const content = {
+  search: 'Serum uric acid concentration',
+  point: '1. Erythrocyte Sedimentation Rate',
+};
+const modifiers = '';`;
+
+const defaultSlotCode = `<template>
+  ${defaultSlotTemplate}
+</template>
+
+<script setup lang="ts">
+${defaultSlotScript}
+</script>`;
+
+const { argTypes } = useArgTypes(UiTabs);
+
+const meta = {
+  title: 'Organisms/Tabs',
+  component: UiTabs,
+  args: {
+    modelValue: 'point',
+    items: complexItemsData,
+    content: {
+      search: 'Serum uric acid concentration',
+      point: '1. Erythrocyte Sedimentation Rate',
+    },
+    modifiers: [],
```

Comment:
should be handled by useArgsTypes; If it doesn't that means useArgsTypes has some issue to solve; I will check it.
```suggestion
```